client/sites/tools/staging-site/components/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -360,6 +361,110 @@ const SyncCardContainer = ( {
 	const siteSlug = useSelector( getSelectedSiteSlug );
 	const isJetpackConnectionError = useIsJetpackConnectionSyncError( error );
 	const isFailedSyncError = useIsFailedSyncError( error );
+	const hasEnTranslation = useHasEnTranslation();
+	const hasNewTranslations = [
```

Comment:
I have decided to check all translations at once to simplify the logic and make it easier to remove the language fallback once the translations are ready.